internal/controller/operatorconfigmap_controller.go

```diff
@@ -762,6 +767,23 @@ func (c *OperatorConfigMapReconciler) reconcileRecipeOperatorSubscription() erro
 	return nil
 }
 
+func (c *OperatorConfigMapReconciler) reconcileODFSnapshotterSubscription() error {
+	odfSnapshotterSubscription, err := c.getSubscriptionByPackageName("csi-snapshotter")
+	if kerrors.IsNotFound(err) {
+		odfSnapshotterSubscription, err = c.getSubscriptionByPackageName("odf-csi-snapshotter")
```

Comment:
@Nikhil-Ladha i think we dont need this check, we always use `odf-csi-snapshotter`

Reply:
Ack, updated.